packages/e2e-tests/test/e2e-current-op.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { expect } from 'chai';
+import {
+  skipIfApiStrict,
+  startSharedTestServer,
+} from '../../../testing/integration-testing-hooks';
+import { TestShell } from './test-shell';
+
+describe('e2e currentOp', function () {
+  skipIfApiStrict();
```

Comment:
I kept it as most e2e had it but this can be removed right? not too familiar with full implications of the apiStrict option

Reply:
We just add this for tests that cover features which aren't part of the stable API – and $currentOp, in particular, is not part of that.

Fwiw I think this is also something that could be added to the existing tests in `e2e.spec.ts`, unless we expect to add more tests for currentOp-specific behavior in the long run